parm/templates/stage.yaml

```diff
@@ -1,11 +1,11 @@
-window begin: '{{window_begin}}'
-window length: $(window_length)
+window begin: '{{ window_begin }}'
+window length: {{ window_length }}
```

Comment:
The marine prep is failing at that line. Is that the correct translation to jinga?

Reply:
what is the error message? it should be the correct Jinja translation.

Reply:
``` 
in "/work2/noaa/stmp/cmartin/CI/GDASApp/workflow/PR/928/global-workflow/sorc/gdas.cd/parm/templates/stage.yaml", line 2, column 16
found unhashable key
```
... humm ... maybe a type issue?

Reply:
is it parsing it as YAML or as Jinja?

Reply:
I think we're the only ones left using that file, could we just revert the changes? We'll remove our dependency on this file early this week.

Reply:
> is it parsing it as YAML or as Jinja?

Ha, it's using YAMLFile from wxflow.

Reply:
I'm fine with just waiting until the dependency is removed. This PR has been open for a while, what's another couple of days.

Reply:
@CoryMartin-NOAA , we need to revert the changes made to the soca yamls. The current plan is to remove the dependency on YAMLFile when we refactor our scripts.

Reply:
@guillaumevernieres yes, I will do so later today